packages/react/src/components/Pagination/Pagination.tsx

```diff
@@ -6,27 +6,25 @@ import IconButton from '../IconButton';
 import TooltipTabstop from '../TooltipTabstop';
 import Icon from '../Icon';
 
+export type StatusLabeller = (
+  totalItems: number,
+  itemStart: number,
+  itemEnd: number
+) => React.ReactNode;
```

Comment:
My preference here would still be to allow for component props to be used here as opposed to a pure function.

```ts
type StatusLabeler = ({
  totalItems: number,
  itemStart: number,
  itemEnd: number
}) => React.ReactNode;
```

I feel that props gives greater flexibility since it's a little easier to destructure props by name as opposed to the property order in the function.